src/anm/graphics/path.js

```diff
@@ -62,6 +63,10 @@ function Path(val) {
         this.segs = val;
     }
 
+    if (engine.Path2D) {
+        this.path2d = new Path2D(Path.toSVGString(this));
```

Comment:
most likely the path will be created from importer (which takes binary path data from the json) - so the SVG string will not be available here. we can rig the importer to convert binary path to svg path though - this way we can drop this line and some others too.